bin/generate-showcase-files.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const path = require('path');
+const pretty = require('js-object-pretty-print').pretty
+const dextroseExtension = '.dextrose.tmp.js';
+const showcaseExtensionFilesToFind = '.showcase.js';
+
+const createDextroseTmp = (dirPath, files) =>
+  new Promise((resolve, reject) => {
+    files.forEach((file) => {
+      const showcaseObject = require(path.resolve(dirPath, file)).default
+      showcaseObject.children = showcaseObject.children.filter(currentValue => currentValue.type.includes('story')); 
+      
+      fs.readFile(file, 'utf8', (err, data) => {
+        if (err) throw err;
+
+        const showcaseComponentImports = data.split('export')[0];
+        const indentSpacing = 4;
+        const showCaseCompoentObjectString = pretty(showcaseObject, indentSpacing, 'PRINT', true);
+
+        const fileToWrite = `${showcaseComponentImports}export default  ${showCaseCompoentObjectString}`;
+
+        const fileName = file.split('.js')[0];
```

Comment:
Might be worth creating a file utility class to abstract some of the implementation away from here. You could end up with 'file.name', 'file.extension' etc then, which would read nicer.